docs/griptape-framework/data/vector-stores.md

```diff
@@ -152,4 +152,62 @@ The following methods are available in the `MarqoVectorStoreDriver` class:
 8. `create_index(self, name: str)`: Creates a new index in the Marqo client with the given name.
 9. `delete_index(self, name: str)`: Deletes an index in the Marqo client with the given name.
 10. `get_indexes(self)`: Returns a list of all indexes in the Marqo client.
-11. `upsert_vector(self, vector: list[float], vector_id: Optional[str] = None, namespace: Optional[str] = None, meta: Optional[dict] = None)`: Inserts a vector into the Marqo index. If a vector with the given vector ID already exists, it is updated; otherwise, a new vector is inserted. Currently, this function is not implemented and raises an Exception.
\ No newline at end of file
+11. `upsert_vector(self, vector: list[float], vector_id: Optional[str] = None, namespace: Optional[str] = None, meta: Optional[dict] = None)`: Inserts a vector into the Marqo index. If a vector with the given vector ID already exists, it is updated; otherwise, a new vector is inserted. Currently, this function is not implemented and raises an Exception.
+
+
+## MongoDBAtlasVectorStoreDriver
+This driver supports the MongoDB Atlas vector database. In addition to standard vector driver methods, it also provides easy document management with MongoDB's powerful NoSQL document store.
+
+Here is a complete example of how the driver can be used to load and query information from MongoDB Atlas:
+
+```python
+from griptape import utils
+from griptape.drivers import MongoDBAtlasVectorStoreDriver
+from griptape.engines import VectorQueryEngine
+from griptape.loaders import WebLoader
+from griptape.structures import Agent
+from griptape.tools import KnowledgeBaseClient
+import openai
+
+# Set the OpenAI API key
+openai.api_key_path = "../openai_api_key.txt"
+
+# Define the namespace
+namespace = "griptape-ai"
+
+# Initialize the vector store driver
+vector_store = MongoDBAtlasVectorStoreDriver(
```

Comment:
Lower case "d" :)

Reply:
Sorry, that was the older version i was doing yesterday, changed it with the new documentation!